lucene/core/src/java/org/apache/lucene/search/FuzzyQuery.java

```diff
@@ -146,6 +147,10 @@ public boolean getTranspositions() {
     return transpositions;
   }
 
+  public Automaton getAutomaton() {
```

Comment:
The name of this method is totally trappy.  We are in fact *building* an automaton... or if you prefer toAutomaton.

Reply:
++, changed